frankenphp.c

```diff
@@ -860,11 +858,7 @@ int frankenphp_request_startup() {
   if (php_request_startup() == SUCCESS) {
     return SUCCESS;
   }
-
```

Comment:
It's not immediately clear if this is needed since we also reset at the beginning of a request.